todolist/src/components/main/to-do-list-components/templates/options-select.vue

```diff
@@ -0,0 +1,10 @@
+<template>
+  <option disabled value="">Please select one</option>
+  <option>Life</option>
+  <option>Education</option>
+  <option>Work</option>
+  <option>Nature</option>
+  <option>Others</option>
+</template>
+
+<script setup lang="ts"></script>
```

Comment:
Okay lang ni kuys? Correct me if my interpretation is wrong/unnecessary.

 Actually, I used this vue-file as my category-options. I separated this part mainly because I reused these very specific options, and if I need to change/add/remove options, I could just refer to this file.

Reply:
Noted kuys